package.json

```diff
@@ -14,7 +14,7 @@
     "build": "cross-env NODE_ENV=production webpack --mode production",
     "build-alias": "cross-env USE_ALIAS=true NODE_ENV=production webpack --mode production",
     "release-heroku": "babel-node ./tools/release-heroku.js",
-    "start": "webpack-dev-server --mode development --open",
+    "start": "cross-env NODE_ENV=development webpack-dev-server --mode development --open",
```

Comment:
Why did we need to add this?

I'm asking because most of webpack 4 is supposed to be smarter about this sort of thing with `mode`.

Reply:
I dont think the dev server is there yet. Without it,
It is running with empty node env. Here I’m just making sure we are running with that env variable set. I don’t think it hurts. Specially because we are relying on that in the webpack config